docker_images/mongodb-3.0.5_tiger/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# MongoDB (gewo/mongodb)
+FROM gewo/mongodb-base
```

Comment:
you can build `FROM gewo/mongodb:3.0.5` and then just add the new start_mongodb script